std/math/hardware.d

```diff
@@ -905,7 +905,18 @@ private:
     // Read from the control register
     package(std.math) static ControlState getControlState() @trusted pure
     {
-        version (LDC)
+        version (RISCV_Any)
+        {
+            mixin(`
```

Comment:
why is a mixin used here?

Reply:
Thanks for your review. I'm not so sure why the previous implementation uses mixin. All I've done is moving this `RISCV_Any` condition above the `LDC` condition. It was first committed in https://github.com/dlang/phobos/commit/3d84369f87be8a341772f5356569500fdf25aba1.

Reply:
IIRC, that was a problem of the dscanner linter not supporting GDC-style inline asm...

Reply:
ah ok. Well it's OK. Refactoring is usually better done separately :)